gems/quilt_rails/lib/quilt_rails/react_renderable.rb

```diff
@@ -8,11 +8,16 @@ def render_react
       url = "#{Quilt.configuration.react_server_protocol}://#{Quilt.configuration.react_server_host}"
       ReactRenderable.log("[ReactRenderable] proxying to React server at #{url}")
 
-      reverse_proxy(url) do |callbacks|
-        callbacks.on_response do |status_code, _response|
-          ReactRenderable.log("[ReactRenderable] #{url} returned #{status_code}")
+      begin
+        reverse_proxy(url) do |callbacks|
+          callbacks.on_response do |status_code, _response|
+            ReactRenderable.log("[ReactRenderable] #{url} returned #{status_code}")
+          end
         end
+      rescue Errno::ECONNREFUSED => e
+        raise "[ReactRenderable] Errno::ECONNREFUSED: Waiting for React server to boot up. If this error presists verify that @shopify/react-server is configured on #{url}"
```

Comment:
Should I drop the `[ReactRenderable]` prefixing

Reply:
/shrug maybe just `[Quilt]` or something

Reply:
I feel like `[Quilt]` is too vague, especially to someone who might not know what quilt is who's just trying to get an app setup.

Reply:
then keep it like this :p